uast/uast.go

```diff
@@ -405,6 +405,34 @@ const (
 	// (e.g. Visual Basic).
 	This
 
+	// List/Map/Set comprehensions are expressions that iterate over a container, optionally
+	// using restricting conditions, and generate a new List/Dict/Set with a subset of the
+	// elements of the original container optionally with some transformation applied to
+	// every element. These are parent nodes where the expression sublements will be mapped
+	// on the Comprehension* roles.
+
+	// Parent node for List comprehensions
+	ListComprehensionExpression
+	// Parent node for Map comprehensions
+	MapComprehensionExpression
+	// Parent node for Set comprehensions
```

Comment:
Thank you. Man, I should write a Vim plugin or something to catch these, I don't know why this mania of mine of skipping final dots in comments that translates to generated documentation in this case,.